cmd/config-v10.go

```diff
@@ -45,6 +45,7 @@ type aliasConfigV10 struct {
 	API          string `json:"api"`
 	Path         string `json:"path"`
 	License      string `json:"license,omitempty"`
+	APIKey       string `json:"api_key,omitempty"`
```

Comment:
```suggestion
	APIKey       string `json:"apiKey,omitempty"`
```

It is preferred to use camelCase style in `mc`